components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -81,6 +81,22 @@ Refer to the following pages on how to create users and assign roles in ServiceN
 - [Creating Users in ServiceNow](https://docs.servicenow.com/csh?topicname=t_CreateAUser.html)
 - [Assigning Roles in ServiceNow](https://docs.servicenow.com/csh?topicname=t_AssignARoleToAUser.html)
 
+### Discovery Source
+
+The Chef Automate Integration App provides a unique ID as a discovery source that distinguishes between discovered CIs and imported CIs. The **Discovery Source** field is part of the `cmdb_servers` table, and it takes **ChefAutomate** value while importing CIs by default.
+
+{{< figure src="/images/automate/snow_integration_plugins.png" alt="Discovery Source">}}
+
+{{< note >}}
+
+The servers recognized by Chef Automate Integration App have discovery source value as **ChefAutomate**.
+
+{{< /note >}}
+
+Refer [Add New Discovery Source](https://developer.servicenow.com/connect.do#!/event/knowledge19/LAB0577/knowledge19_LAB0577_add_new_discovery_source) page to create a new and unique discovery source that distinguishes discovered CIs and imported CIs.
```

Comment:
```suggestion
Refer to ServiceNow's [Add New Discovery Source](https://developer.servicenow.com/connect.do#!/event/knowledge19/LAB0577/knowledge19_LAB0577_add_new_discovery_source) page to create a new and unique discovery source that distinguishes discovered CIs and imported CIs.
```

Reply:
This reference is removed based on Eng. team comments